Dockerfile

```diff
@@ -12,13 +12,11 @@
 WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
+ENV ENV_ENGINE="default"
+ENV ENV_NAME="default"
+ENV ENV_USER="default"
+ENV ENV_PASSWORD="default"
+ENV ENV_HOST="default"
```

Comment:
The environment variable `ENV_HOST` is set to 'default'. Ensure that this default value is appropriate for your application, or consider overriding it with a more specific value during deployment.